docs/pages/feedback/firebase-admin.md

```diff
@@ -61,12 +61,16 @@ if (!admin.apps.length) {
 export default admin.firestore();
 ```
 
+<Callout>
+By default, Firebase sets your databaseURL to `'https://your-app-id.firebaseio.com'`
+</Callout>
+
 ## Retrieving Data
 
-Finally, lets create `lib/db-admin.js` to fetch data from Firestore.
+Finally, lets create `utils/db-admin.js` to fetch data from Firestore.
 This function will fetch all `Feedback` where the `siteId` matches.
 
-**`lib/db-admin.js`**
+**`utils/db-admin.js`**
```

Comment:
I understand the final project has those files in `lib/`, however, in the docs, we're using `utils/`, so I decided to update it here for consistency.

Reply:
I would think it should be the inverse. All places in the docs should reference `lib` now.

Looking at the file above in this code review, I see:

```
import { getAllFeedback, getAllSites } from '@/lib/db-admin'
```

Reply:
This should do the trick https://github.com/leerob/react2025/pull/7/commits/246c918cf84877926ca5bd4743c842b8fa7cea0a